### ISSUE TITLE
Initialization of house mode causes runtime error at start

### ISSUE THREAD

**Issue by toggledbits** [toggledbits/Reactor#6]
A missing parameter from the (attempted) initialization of house mode (on the Reactor device) causes a Lua failure at startup for new installers.

**Comment by toggledbits** [comment]
The temporary workaround is to submit the following Lua code in _Apps > Develop Apps > Test Luup Code_:

```
reactor_device = <put your reactor device number here>
luup.variable_set( "urn:toggledbits-com:serviceId:Reactor", "Version", 100, reactor_device )
luup.reload()

```

After executing this code, Reactor should start up normally. Permanent fix is slated for v1.4

**Comment by toggledbits** [comment]
Also fixed in stable branch.

**Comment by toggledbits** [comment]
Fixed in v1.4
